lib/runner.js

```diff
@@ -235,6 +235,9 @@ Runner.prototype.fail = function(test, err) {
  */
 
 Runner.prototype.failHook = function(hook, err){
+  if(hook.parent && hook.parent.file){
```

Comment:
Will do!  Not sure how to squash the commit, because the commit adding the title is by a different author.  

The way I got into this was by having the same issue, I saw the other author's fix, but noticed it didn't have file support.